rust/src/cli/apply.rs

```diff
@@ -60,52 +62,34 @@ where
     } else {
         DEFAULT_TIMEOUT
     };
-    let mut content = String::new();
-    // Replace non-breaking space '\u{A0}'  to normal space
-    reader.read_to_string(&mut content)?;
-    let content = content.replace('\u{A0}', " ");
-
-    let mut net_state: NetworkState = match serde_yaml::from_str(&content) {
```

Comment:
There are 37 changed files, this is hard to review properly. Can you please split into several commits so they can be reviewed in smaller logical steps? Also, consider that a single PR might need to be reviewed more than once, so it gets even harder.